vpr/src/route/check_route.cpp

```diff
@@ -625,6 +627,23 @@ static void check_node_and_range(int inode, enum e_route_type route_type) {
     check_rr_node(inode, route_type, device_ctx);
 }
 
+//Checks that all non-configurable edges are in a legal configuration
+//This check is slow, so it has been moved out of check_route()
+static void check_all_non_configurable_edges() {
+    VTR_LOG("\n");
```

Comment:
Rather than these VTR_LOG's, use the scoped timer?

Reply:
What about a progress indicator? This check is really slow. I don't want to add log spam, though.

Reply:
For now, lets just have the runtime cost, so we can easily compare changes (e.g. bitset).  I think a progress bar would lead to logspam.

Reply:
Done.